database/redis/selfstate.go

```diff
@@ -47,28 +47,43 @@ func (connector *DbConnector) GetRemoteChecksUpdatesCount() (int64, error) {
 }
 
 // GetNotifierState return current notifier state: <OK|ERROR>
-func (connector *DbConnector) GetNotifierState() (string, error) {
+func (connector *DbConnector) GetNotifierState() (string, string, error) {
 	c := connector.pool.Get()
 	defer c.Close()
-	ts, err := redis.String(c.Do("GET", selfStateNotifierHealth))
-	if err == redis.ErrNil {
-		ts = moira.SelfStateOK
-		err = connector.SetNotifierState(ts)
-	} else if err != nil {
-		ts = moira.SelfStateERROR
+
+	var state, message string
+	stateArgs := []interface{} {selfStateNotifierHealth, selfStateNotifierMessage }
+	values, err := redis.Strings(c.Do("MGET", stateArgs...))
```

Comment:
I've used `MGET` (and `MSET` for SetNotifierState too) as it feels better than getting State and Message separately and using multiple if-statements to check for their errors. I'm not so confident about it though, and would like to know what you think.